test/parse.js

```diff
@@ -236,6 +236,30 @@ test('circuit original -> parse - > stringify -> sorted original', t => {
 	t.deepEqual(queryString.stringify(queryString.parse(original, options), options), sortedOriginal);
 });
 
+test('circuit parse -> stringify with array commas', t => {
```

Comment:
2nd step: In string to object conversion, some definition is lost. The string contains empty values, with no information about empty string or null. The object cannot be converted back to what it was in step1. Ok?

Reply:
That's fine as long as the behavior is documented.